src/output-json-dhcp.c

```diff
@@ -78,7 +78,7 @@ static int JsonDHCPLogger(ThreadVars *tv, void *thread_data,
         return TM_ECODE_FAILED;
     }
 
-    json_t *dhcp_js = rs_dhcp_logger_log(ctx->rs_logger, tx);
+    json_t *dhcp_js = (json_t *)rs_dhcp_logger_log(ctx->rs_logger, tx);
```

Comment:
can we avoid these casts?

Reply:
Looks like we could do something like this in `cbindgen.toml`:
```
[export]
exclude = ["JsonT"]

[export.rename]
"JsonT" = "json_t"
```

Reply:
that worked